### DISCUSSION TITLE
Quale licenza scegliere?

### DISCUSSION THREAD

**Discussion by francescobianco** [Il-Libro-Open-Source/book#7] [Salotto]
La scelta della licenza è sempre un punto fondamentale da affrontare per qualsiasi prodotto open source, come sapete il panorama delle licenze open è variegato. 

Immagino che anche questo sarà un elemento che gli ambassador dovranno gestire.

**Comment by nicolaerario** [comment]
Se non si trattasse di un libro, proporrei una [GNU GPLv3](https://choosealicense.com/licenses/gpl-3.0/);
In questo caso potrebbe essere più opportuna una `Creative Commons`? (direi [CC-BY-SA-4.0](https://choosealicense.com/licenses/cc-by-sa-4.0/)).

**Comment by Cadienvan** [comment]
Ottimo argomento che sicuramente verrà discusso.
Credo che siamo tutti d'accordo sul concetto di "quanto più sarà aperta questa licenza, tanto più si potrà abbracciare l'idea intrinseca del progetto".

**Reply by nicolaerario** [reply]
Per questo motivo ho consigliato la [CC-BY-SA-4.0](https://choosealicense.com/licenses/cc-by-sa-4.0/) che specifica la condizione di `Same license`. Il libro nascerà libero per rimanerci.

**Reply by Cadienvan** [reply]
Sembra una direzione più che interessante!! Grazie Nicola!

**Comment by eppak** [comment]
Aggiungo un commento magari precoce, ma che voi sappiate ne esiste una che contempli la protezione da IA come il training?

**Comment by AngeloAvv** [comment]
@nicolaerario temo che la CC-BY-SA sia troppo permissiva dal punto di vista commerciale (chiunque può prendere il testo, stamparlo e trarne profitto). Piuttosto userei una CC-BY-SA-NC.

Se il lavoro è gratis, che lo sia fino al midollo.

**Comment by AngyDev** [comment]
Ciao a tutt*,

non mi ero accorta che ci fosse una discussione dedicata per la scelta della licenza e avevo quindi fatto un commento nella #3. Riporto qui la parte relativa ad un mio suggerimento

Penso che forse la più appropriata possa essere la [Creative Commons Attribution Share Alike 4.0 International](https://choosealicense.com/licenses/cc-by-sa-4.0/), mantiene le libertà più importanti di utilizzo, modifica e distribuzione ma richiede di dare credito all'autore e di rilasciare i progetti derivati sotto la stessa licenza. Se quest'ultima clausola è troppo restrittiva allora la [Creative Commons Attribution 4.0 International](https://choosealicense.com/licenses/cc-by-4.0/).
Un'alternativa alla prima ma più restrittiva è la GNU Free Documentation License v1.3 che vedo un po' troppo rigida, ma comunque penso sia il caso di citare.

Una nota che vorrei aggiungere riguarda anche eventualmente il copyright dell'opera. Chi lo detiene? Tutti gli ambassador o qualcuno in particolare? Anche questo potrebbe essere un aspetto da valutare soprattutto se magari poi le cose dovessero cambiare in un futuro e qualcuno volesse anche cambiare la licenza.

**Reply by Cadienvan** [reply]
Sono d'accordo sulla Share Alike, se vogliamo che l'opera sia aperta è anche giusto che chi ne faccia riferimento lo faccia in un contesto identico.

Riguardo al copyright, potrei prenderla un po' sottogamba ma la reputo una sottigliezza, nel senso che una volta che l'opera è pubblica e rimane tale (Magari si potrebbe pensare di coinvolgere la CNCF o la Linux Foundation per garantire la cosa) il copyright lascia il tempo che trova.

Anche su questo però sono VERAMENTE ignorante, @AngyDev io ti ricordo che fino a fine mese ci si può proporre qui (#5 ) e una persona con le tue competenze sarebbe una manna dal cielo.

P.S. So che sarai al codemotion e alcuni di noi saranno lì, quindi spero ci sarà modo di parlarci e conoscerci :)

**Reply by AngyDev** [reply]
@Cadienvan fatta candidatura. Grazie per il suggerimento. Sono molto curiosa di intraprendere questa avventura. :) 
Sarò sicuramente presente entrambi i giorni, farò in modo di trovarvi :)

Probabilmente hai ragione per quanto riguarda il copyright, vorrei accertarmi che non possa essere un problema in futuro solo a livello decisionale. È anche vero che stiamo parlando di un libro, quindi una volta presa una decisione non vedo perché cambiarla. Sono d'accordo con l'idea eventualmente di coinvolgere le Fondazioni.

**Comment by rrhinox** [comment]
Premessa, so che sono ignorantissimo su questo tema ma ho scoperto questa fondazione per puro caso : 
https://github.com/EbookFoundation > https://ebookfoundation.org/ > https://www.gitenberg.org/license > https://creativecommons.org/licenses/by-nc/4.0/ e loro usano CC BY-NC 4.0 DEED - Attribution-NonCommercial 4.0 International. (se ho capito bene :P) 
Tra l'altro il tutto nasce dal progetto : https://www.gutenberg.org/ (bloccato dagli ISP Italiani)  
Può esserci utile per valutare in futuro quale licensing usare @Cadienvan ?

**Reply by Cadienvan** [reply]
@AngyDev questo credo sia la conferma di ciò che ci dicevamo l'altro giorno, ovvero che NC non vuol dire senza donazioni.

**Reply by AngyDev** [reply]
Si @Cadienvan, direi che possiamo confermare questa possibilità

**Reply by Cadienvan** [reply]
@rrhinox confermo che quella è una delle licenze che stiamo pensando di adottare!

**Comment by guizzo** [comment]
Segnalo che è in corso una [PR](https://github.com/Il-Libro-Open-Source/book/pull/160) per l'aggiunta della licenza al progetto.

Una volta che sarà dentro, potremo chiudere questa discussione

**Comment by guizzo** [comment]
Discussione risolta nella [PR #160](https://github.com/Il-Libro-Open-Source/book/pull/160)

Licenza disponibile [QUI](https://github.com/Il-Libro-Open-Source/book/blob/main/LICENSE.md)
